### PR TITLE
Edit new/changed content added to Filebeat and changelog for 1.1

### PR DESCRIPTION
**Pull request by dedemorton**
Copy edits for style/consistency.  I wasn't planning to edit the CHANGELOG, but I noticed a typo and decided to give the new info for 1.1 (in the commented out section) a quick edit. 

@monicasarbu The following description in the CHANGELOG is not very end user friendly, and it's hard to follow: "Fix setting direction to out and use its value to decide when dropping events if ignore_outgoing is enabled {pull}557[557]." Can you rephrase this sentence to more accurately reflect how the change affects the user? If you want to suggest a change in the comments here, I can add it and submit the change with this PR. Thanks!

### CODE REVIEW

**Comment by monicasarbu** [comment]
Here is another try: "Fix the condition that decides when the direction of the transaction is set to "outgoing". Use the value of the direction field to decide when dropping the outgoing transactions."

Feel free to change it to sound better :)

**Comment by dedemorton** [comment]
@monicasarbu I changed the description to say: "Fix the condition that determines whether the direction of the transaction is set to 'outgoing'. Packetbeat uses the direction field to determine which transactions to drop when dropping outgoing transactions." Is that correct?

**Comment by monicasarbu** [comment]
Yes, great. Thanks!

**Comment by monicasarbu** [comment]
@dedemorton Can you please rebase? Thanks!

**Comment by dedemorton** [comment]
@monicasarbu #841 should be ready to merge. Closing this old PR.
